dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -1,6 +1,49 @@
 package ExtUtils::ParseXS;
 use strict;
 
+# Note that the pod for this module is separate in ParseXS.pod.
+#
+# This module provides the guts for the xsubpp XS-to-C translator utility.
+# By having it as a module separate from xsubpp, it makes it easier to
```

Comment:
This is an important advantage, but to the best of my knowledge that's not why most of the code was moduled out. The direct cause was making it more usable from Module::Build and friends (so they don't have to shell out).